.solhint.json

```diff
@@ -12,6 +12,7 @@
         "no-inline-assembly": "off",
         "quotes": ["error", "double"],
         "no-empty-blocks": "off",
-        "compiler-version": "off"
+        "compiler-version": "off",
+        "func-name-mixedcase": "off"
```

Comment:
If you mean the foundry tests I should either update their names or we should switch on snake_case or if that's not possible just switch off mixed case for the tests. mixed case will be important for the remainder of the solution